README.md

```diff
@@ -16,3 +16,98 @@ https://github.com/dwyl/intellectual-property and raise an
 on GitHub if you require clarification!
 
 See: https://github.com/dwyl/contributing/issues/1 + https://github.com/dwyl/summer-2015/issues/27
+
+## Recommended Guidelines
+
+### Issues
+
+At dwyl we use _issues_ to log our user stories and technical tasks for a given project. User stories are higher level issues that describe certain functionalities of the application you're building. These user stories get broken down into smaller **_technical_** tasks.
+
+### Issues > Before You Start
+
+Before you begin any work on a dwyl project please open an issue on the repository that you wish to contribute to.
+
+There **_must_** be a **comment** from **at least one other person** on the issue acknowledging it and/or confirming the requirement/bug before work starts. This is necessary to avoid doing _unnecessary work_. e.g. someone may have already solved that problem or implemented a similar feature on a different project and can point you in the direction of sample code or the PO/Client may deem the issue/story to be lower priority than something else on the list. The recommended workflow is as follows:
+
```

Comment:
I would split this simple recommended workflow into its own section. We want to make contributing _easy_ and if people see a huge readme, they'll want to know what's important. If they only read one thing, this should be it